src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/jira/StandaloneStoppedNode.kt

```diff
@@ -153,6 +153,13 @@ internal data class StandaloneStoppedNode(
                 break
             }
             if (deadline < now()) {
+                if (logger.isDebugEnabled) {
+                    val logContent = ssh.safeExecute(
+                        cmd = "cat $jiraHome/log/atlassian-jira.log",
```

Comment:
It would be great to download all rolled logs, not only latest one as the reason of failure might logged somewhere at the beginning

Reply:
Do you have any suggestion how to do that without changing the whole architecture of the lib?

I personally thought that a proper solution would be to not use Exceptions as signals of failure. This would allow us to control artifact gathering even for infrastructure that would fail to provision. 

Yet for such big changes I think we need to do a lot more work. I see it as a next step, unless there is some easier way then the logging or change of Exception APIs.

Reply:
Simple cat command change should be enough
```echo 0 > test.log
echo 1 > test.log.1
echo 2 > test.log.2
cat test.log*
0
1
2

Reply:
Is the order of files in such print guaranteed?

I believe Jira's logs roll so that higher number at the end means older log file. If so we would probably want it to be printed the other way around to avoid confusion.

Reply:
I'm not sure how it will behave on Jira logs, needs to be tested on the Jira host

Reply:
I see that it would take more of my time to ship this. I will stick to printing only one file for now. Even with printing all of them we won't get full log history, as Jira deletes the oldest files.

We can add support for printing all `atlassian-jira.log*` files later on at any point in the future without breaking anything.

Reply:
Having the last log file is still better than having nothing. Agreed, we can improve it later